src/atomate2/qchem/flows/core.py

```diff
@@ -0,0 +1,169 @@
+"""Core QChem Flows"""
+
+from copy import deepcopy
+from dataclasses import dataclass, field
+from pathlib import Path
+
+from jobflow import Flow, Maker
+from pymatgen.core.structure import Molecule
+
+from atomate2.qchem.jobs.base import BaseQChemMaker
+from atomate2.qchem.jobs.core import (
+    OptMaker,
+    TransitionStateMaker,
+    FreqMaker,
+)
+
+
+# from atomate2.qchem.schemas.calculation import VaspObject
+# from atomate2.vasp.sets.core import HSEBSSetGenerator, NonSCFSetGenerator
+
+__all__ = [
+    "FrequencyFlatteningOptimizeMaker",
+    "FrequencyFlatteningTransitionStateMaker",
+]
+
+
+@dataclass
+class FrequencyFlatteningOptimizeMaker(Maker):
+
+    #need to incorporate iterations and perturb geometry
+    """
+    Maker to iteratively optimize given structure and flatten imaginary frequencies to ensure that
+    the resulting structure is a true minima and not a saddle point.
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this maker.
+    freq_maker : .BaseQChemMaker
+        Maker to generate the frequency calculation.
+    opt_maker : .BaseQChemMaker
+        Maker to generate the optimization calculation.
+    """
+
+    name: str = "frequency flattening structure optimization"
+    max_iterations=10
```

Comment:
For dataclasses you need to specify the type. E.g., this should be
```suggestion
    max_iterations: int = 10
```

Otherwise the dataclass won't work properly.

Please double check all your dataclasses.